### ISSUE TITLE
Make start of the week configurable

### ISSUE THREAD

**Issue by bakua** [jama5262/jiffy#64]
Hello,

it would be nice to be able to configure start of the week to override locale's default. I have an app that manipulates time and this is an important part of an app settings for me.

Thank you

**Comment by jama5262** [comment]
@bakua Yes, this is an important feature that is demanded. Planned on adding it soon. There is a similar issue you can also view, see #58

**Comment by bakua** [comment]
Hello, may I ask why this issue is closed? I don't see any code updates in repo that would fix this issue. Thank you.

**Comment by Adrian4777** [comment]
Does anyone know if this is implemented yet? Jiffy().week is giving me the wrong week, I would like to use Monday-Sunday.

**Comment by krwrang** [comment]
> @bakua Yes, this is an important feature that is demanded. Planned on adding it soon. There is a similar issue you can also view, see #58

Define soon?
